src/lib/viewers/controls/media/MediaSettingsControls/MediaSettingsRadioItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classNames from 'classnames';
+import IconCheckMark24 from '../../icons/IconCheckMark24';
+import { decodeKeydown } from '../../../../util';
+import './MediaSettingsRadioItem.scss';
+
+export type Props<V extends Value> = {
+    className?: string;
+    isSelected?: boolean;
+    label?: string;
+    onChange: (value: V) => void;
+    value: V;
+};
+export type Ref = HTMLDivElement;
+export type Value = boolean | number | string;
+
+function MediaSettingsRadioItem<V extends Value>(props: Props<V>, ref: React.Ref<Ref>): JSX.Element {
+    const { className, isSelected, label, onChange, value } = props;
+
+    const handleClick = (): void => {
+        onChange(value);
+    };
+
+    const handleKeydown = (event: React.KeyboardEvent<Ref>): void => {
+        const key = decodeKeydown(event);
+
+        if (key !== 'ArrowLeft' && key !== 'Enter' && key !== 'Space') {
+            return;
+        }
+
+        onChange(value);
+    };
+
+    return (
+        <div
+            ref={ref}
+            aria-checked={isSelected ? 'true' : 'false'}
+            className={classNames('bp-MediaSettingsRadioItem', className, {
+                'bp-is-selected': isSelected,
+            })}
+            onClick={handleClick}
+            onKeyDown={handleKeydown}
+            role="menuitemradio"
+            tabIndex={0}
+        >
+            <div className="bp-MediaSettingsRadioItem-check">
+                <IconCheckMark24 className="bp-MediaSettingsRadioItem-check-icon" height={16} width={16} />
+            </div>
+            <div className="bp-MediaSettingsRadioItem-value">{label || value}</div>
```

Comment:
What is the purpose of having both `label` and `value` for this component?

Reply:
It will be relevant for the "rate" menu, which has values that are their own label, such as `1.5`.